src/main/java/homework3/ex2/page/Ex2LeftNavigationBar.java

```diff
@@ -0,0 +1,65 @@
+package homework3.ex2.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class Ex2LeftNavigationBar extends BasePage {
+
+    public Ex2LeftNavigationBar(WebDriver driver) {
```

Comment:
According to Java code convention methods must be lower than fields. Plus should be order:

fields
constructor (if present)
methods

Fix pls